### DISCUSSION TITLE
Press Material/Flyers info and images

### DISCUSSION THREAD

**Discussion by MTRNord** [streetcomplete/StreetComplete#4500] [Q&A] [answered]
Hi, I was wondering if there is any press/flyers information available for streetcomplete. I am aware of general OSM materials, however I consider streetcomplete to be much more entry level than generic OSM. So I was wondering if anybody already has something for getting beginners on board. :) Mostly, I am looking for a thing I can lay out in my local library or similar places to get some new people into OSM.

If nothing like that was done yet:

Are there Logos/Images that we can use to create this info material as well as maybe what the maintainers wish to be listed in the information of such document. :)

**Comment by matkoniecz** [comment, marked as answer]
I am not aware and neither https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#streetcomplete-related-projects nor https://github.com/topics/streetcomplete lists anything

If you publish repo with such materials on open source license, then it would be worth listing there

> maybe what the maintainers wish to be listed in the information of such document. :)

- have fun while mapping!
- StreetComplete is for mapping while surveying

**Reply by MTRNord** [reply]
I opened https://github.com/MTRNord/StreetCompleteFlyer now. It is just some blocking text for now I wrote down as it came to mind in the last 30m and I will improve it over the next few hours. Feel free to already add comments via issues though if there is already a major misconception in this so I can fix this early. :) The whole thing for now is meant to be a single page flyer to layout at local stores or libraries and to be picked up by those that do not want to use the web editor or spend much time in the wiki. It is also designed to be specifically promoting streetcomplete considering from my experience this is so far the easiest way for newcomers to work with OSM

**Reply by MTRNord** [reply]
(also just to clarify: the current document is just filled with basically notes of thoughts I had. It likely needs a few more days to be an actual flyer you can print/use)

**Reply by westnordost** [reply]
Note there is also https://streetcomplete.app now , with current screenshots etc, available in many languages

**Reply by MTRNord** [reply]
Awesome! That seems like a great place to point people at for further info :) because in the end a flyer sadly doesn't have unlimited space :D
